components/Inventory/basic_usage.rst

```diff
@@ -0,0 +1,272 @@
+
+.. note::
+    For more detailed information go to `Sylius API NoopInventoryOperator`_.
+
+.. _Sylius API NoopInventoryOperator: http://api.sylius.org/Sylius/Component/Inventory/Operator/NoopInventoryOperator.html
+
+BackordersHandler
+-----------------
+
+The **BackorderHandler** changes inventory unit state.
+
+Process backorders
+~~~~~~~~~~~~~~~~~~
+
+This method will change inventory unit state to ``backordered`` if quantity of requested inventory units will be insufficient.
+
+.. code:: php
+
+    <?php
+
+    use Sylius\Component\Inventory\Operator\BackordersHandler;
+    use Doctrine\Common\Collections\ArrayCollection;
+    use Sylius\Component\Inventory\Model\InventoryUnit;
+    use Sylius\Component\Inventory\Model\InventoryUnitInterface;
+
+    $inventoryUnitRepository; //Repository model.
+    $stockable; //Stockable model.
+
+    $backordersHandler = new BackordersHandler($inventoryUnitRepository);
+    $inventoryUnit1 = new InventoryUnit();
+    $inventoryUnit2 = new InventoryUnit();
+    $inventoryUnits = new ArrayCollection();
+
+
+    $stockable->getOnHand(); //Output will be 1.
+    $inventoryUnit1->setStockable($stockable);
+    $inventoryUnit1->setInventoryState(InventoryUnitInterface::STATE_SOLD);
+
+    $inventoryUnit2->setStockable($stockable);
+    $inventoryUnit2->setInventoryState(InventoryUnitInterface::STATE_CHECKOUT);
+
+    $inventoryUnits->add($inventoryUnit1);
+    $inventoryUnits->add($inventoryUnit2);
+    count($inventoryUnits); //Output will be 2.
+
+    $backordersHandler->processBackorders($inventoryUnits);
+
+    $inventoryUnit2->getInventoryState(); //Output will be 'backordered'
+
+Fill backorders
+~~~~~~~~~~~~~~~
+
+This method will change inventory unit state to ``sold``.
+
+.. code:: php
+
+    <?php
+
+    use Sylius\Component\Inventory\Operator\BackordersHandler;
+
+    $inventoryUnitRepository; //Repository model.
+    $stockable; //Stockable model.
+
+    $backordersHandler = new BackordersHandler($inventoryUnitRepository);
+    $stockable->getOnHand(); //Output will be 6.
+
+    //Let assume that we got 9 inventory units with state 'backordered'.
+    //This method will find all inventory units for that specific stockable with 'backordered' state.
+    $backordersHandler->fillBackorders($stockable);
+    // Now 6 of them will have 'sold' state.
+
+    $stockable->getOnHand(); //Output will be 0.
+
+.. note::
+    For more detailed information go to `Sylius API BackordersHandler`_.
+
+.. _Sylius API InventoryUnitFactory: http://api.sylius.org/Sylius/Component/Inventory/Operator/BackordersHandler.html
+
+
+AvailabilityChecker
+-------------------
+
+The **AvailabilityChecker** checks availability for given stockable object.
+To charactrize an object which is an **AvailabilityChecker**, it needs to implement the :ref:`component_inventory_checker_availability_checker_interface`_.
+Second parameter in ``->isStockSufficient()`` method indicates that customer want a merchandise in specific quantity.
```

Comment:
That sentence is a bit too sophisticated imo :)

`Second parameter of the isStockSufficient() method gives a possibility to check for a given quantity of a stockable.`